src/hb-paint.cc

```diff
@@ -30,9 +30,9 @@
 
 static void
 hb_paint_push_transform_nil (hb_paint_funcs_t *funcs, void *paint_data,
-                             float xx, float xy,
```

Comment:
Humm. I know this is controvertial. But I'm used to "xx, xy, yx, yy" order. I think cairo's one of the few to go against that.

Reply:
Nevermind.